PSReadLine/Words.cs

```diff
@@ -94,6 +94,57 @@ private bool InWord(int index, string wordDelimiters)
             return !char.IsWhiteSpace(c) && wordDelimiters.IndexOf(c) < 0;
         }
 
+        private bool InWord(string text, int index, string wordDelimiters)
```

Comment:
Logic is duplicated with the method above. The method above should call this method instead.

Reply:
I refactored this method to directly take a char, and use it in the above method.